.github/workflows/npm-release.yml

```diff
@@ -0,0 +1,22 @@
+name: Publish Package to npmjs
+on:
+  push:
+    tags:
+      - '^v[0-9]+(\.[0-9]+)*([-\.]\S+)?'
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      # Setup .npmrc file to publish to npm
+      - uses: actions/setup-node@v3
```

Comment:
You can use the `.nvmrc` to select the node version. See the `tests` action

Reply:
Pushed some updates which reuses a git form there